server/src/test/java/org/elasticsearch/action/admin/indices/create/SplitIndexIT.java

```diff
@@ -367,7 +367,7 @@ private static IndexMetaData indexMetaData(final Client client, final String ind
 
     public void testCreateSplitIndex() {
         internalCluster().ensureAtLeastNumDataNodes(2);
-        Version version = VersionUtils.randomVersionBetween(random(), Version.V_6_0_0_rc2, Version.CURRENT);
+        Version version = VersionUtils.randomVersionBetween(random(), Version.V_7_0_0, Version.CURRENT);
```

Comment:
This is the kind of test I think should be updated to not use a hardcoded version. Please look at `VersionUtil.randomIndexCompatibleVersion`